.formatter.exs

```diff
@@ -1,4 +1,5 @@
+# Used by "mix format"
 [
-  inputs: ["mix.exs", "{config,lib,test}/**/*.{ex,exs}"],
```

Comment:
Well, this last comma should not be removed. Now `mix format` fails with an error:
```
** (SyntaxError) .formatter.exs:3:3: syntax error before: import_deps
    (elixir 1.11.3) lib/code.ex:340: Code.eval_string_with_error_handling/3
    (mix 1.11.3) lib/mix/tasks/format.ex:310: Mix.Tasks.Format.eval_file_with_keyword_list/1
    (mix 1.11.3) lib/mix/tasks/format.ex:172: Mix.Tasks.Format.eval_dot_formatter/1
    (mix 1.11.3) lib/mix/tasks/format.ex:141: Mix.Tasks.Format.run/1
    (mix 1.11.3) lib/mix/task.ex:394: Mix.Task.run_task/3
    (mix 1.11.3) lib/mix/cli.ex:84: Mix.CLI.run_task/2
```